src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-details/nfs-details.component.html

```diff
@@ -1,11 +1,16 @@
-<tabset *ngIf="selection">
-  <tab heading="Details"
+<tabset *ngIf="selection"
+        stateId="nfs">
+  <tab cdTabStateful
+       stateId="details"
+       heading="Details"
        i18n-heading>
     <cd-table-key-value [data]="data">
     </cd-table-key-value>
   </tab>
 
-  <tab heading="Clients ({{ clients.length }})"
+  <tab cdTabStateful
+       stateId="clients"
+       heading="Clients ({{ clients.length }})"
```

Comment:
for situations where we can't get the id from the heading, we could pass it in the directive attr itself:
```suggestion
  <tab cdTabStateful="clients"
       heading="Clients ({{ clients.length }})"
```

Reply:
That's the reason why i decided against it. Too much alternatives instead of a straigt forward interface using an explicit attribute.

Reply:
There is absolutely no relation between heading and the functionality of the directive. The use of attributes that are intended for a different purpose will sooner or later fall on your feet. Because of that i want to stay on the current implementation by using a dedicated attribute.